src/main/java/bio/terra/cli/command/app/passthrough/Git.java

```diff
@@ -46,15 +54,33 @@ protected void executeImpl() {
     }
     if (cloneAll) {
       validateCloneCommand();
-      Context.requireWorkspace().getResources().stream()
-          .filter(resource -> Resource.Type.GIT_REPO == resource.getResourceType())
-          .forEach(resource -> cloneGitRepoResource(resource));
+      var resources = Context.requireWorkspace().getResources();
+      getGitRepoResourceToClone(resources);
+      resources.stream()
+          .filter(resource -> Resource.Type.DATA_COLLECTION == resource.getResourceType())
+          .forEach(
+              resource -> {
+                Workspace dataCollectionWorkspace =
+                    attemptToGetDataCollectionWorkspace((DataCollection) resource);
+                if (dataCollectionWorkspace != null) {
+                  getGitRepoResourceToClone(dataCollectionWorkspace.getResources());
+                }
+              });
       return;
     }
     if (names != null) {
       validateCloneCommand();
       for (String name : names) {
-        cloneGitRepoResource(Context.requireWorkspace().getResource(name));
+        var resource = Context.requireWorkspace().getResource(name);
+        if (Resource.Type.DATA_COLLECTION == resource.getResourceType()) {
```

Comment:
Interesting... I was just thinking of supporting `terra git clone --all` (have that clone DC git repos) and not `terra git clone --name=1000-genomes`.

1000-genomes isn't a git repo. Could we not support `terra git clone --name=1000-genomes`; if user wants to clone DC git repos, they run `terra git clone --all` ?

Reply:
IMO if you can clone it one way it should be allowed another way